subprojects/docs/src/docs/userguide/dep-man/03-controlling-transitive-dependencies/controlling_transitive_dependencies.adoc

```diff
@@ -53,10 +53,36 @@ Dependency constraints themselves can also be added transitively.
 [[sub:sharing-dep-versions-between-projects]]
 === Sharing dependency versions between projects
 
-// A dependency to a platform inherits all subgraph constraints of the platform (can be turned off by 'doNotInheritStrictVersions()')
+A <<dependency_management_terminology.adoc#sub::terminology_platform,platform>> is a special software component which can be used to control transitive dependency versions.
+In most cases it's exclusively composed of link:#sec:adding-constraints-transitive-deps[dependency constraints] which will either _suggest_ dependency versions or _enforce_ some versions.
```

Comment:
Can you use the `<<link, text>>` notation so that it is picked up by the link checker?

Reply:
Ok I will consolidate in a subsequent PR (I already started working on top of this one).